### ISSUE TITLE
alt text is silently dropped if it overruns the server limit

### ISSUE THREAD

**Issue by dilinger** [elk-zone/elk#1018]
### Pre-Checks

- [X] Check that there isn't [already an issue](https://github.com/elk-zone/elk/issues) that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/elk-zone/elk/discussions) or join our [Discord Chat Server](https://chat.elk.zone).
- [X] Providing a screenshot or video to reproduce the issue or show visually what was meant.
- [ ] I am willing to provide a PR.

### Describe the bug

Using Elk, if alt text in a toot exceeds your instance's limit, the alt text is silently deleted. The toot is published, just with the alt text missing.

### Reproduction video or screenshot

My server's alt text limit is 1500 chars. I pasted in 4200 characters of alt text: 
![Screenshot-2023-01-12,13-11-37](https://user-images.githubusercontent.com/11581730/212147104-0101f634-49de-4263-9e73-eec11a5e4233.png)

After hitting apply, you can see that the alt text badge shows up, and the text is there:
![Screenshot-2023-01-12,13-12-07](https://user-images.githubusercontent.com/11581730/212147199-5fad06e6-d849-493a-824d-a7b5890336e1.png)

But once I hit the publish button, the final toot has no alt text:
![Screenshot-2023-01-12,13-12-39](https://user-images.githubusercontent.com/11581730/212147333-0d1e9bce-ccec-4bcb-bb20-a5d1b76e3721.png)

The alt text is just silently dropped when publishing.

### Additional Context

Mastodon has no API (afaik) for determining the maximum amount of characters for alt text. My instance (social.ridetrans.it) has a limit of 1500 chars. Pinafore (an alternative frontend) arbitrarily enforces a 420 character limit, which is too small in my opinion. Elk enforces no limit at all.

I don't know if the solution here is an arbitrary limit, or to detect an error when publishing and returning the user to the public screen (alt text intact) so they can fix the issue before attempting to once again hit that publish button.

**Comment by ayoayco** [comment]
@dilinger Did we get information when you posted the image with alt text that it exceeded a limit?
